GOVERNANCE.md

```diff
@@ -0,0 +1,158 @@
+# Prometheus Governance v1.0
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Prometheus community. Common terminology used in this governance document are listed below:
+
+* **Team members**: Any members of the private [prometheus-team][team] Google group.
+
+* **Maintainers**: Team members who are leads on an individual project ([MAINTAINERS.md][maintainers.md]).
+
+* **Projects**: Any repository in the Prometheus [GitHub organization][gh].
+
+* **The Prometheus Project**: The sum of all activities performed under this governance, concerning one or more Repositories or the community.
+
+## Values
+
+The Prometheus developers and community are expected to follow the values defined in the [CNCF charter][charter], including the [CNCF Code of Conduct][coc]. Furthermore, the Prometheus community strives for kindness, giving feedback effectively and building a welcoming environment. The Prometheus developer community strives to operate by rough consensus for the majority of decisions and only resort to conflict resolution through a motion if consensus cannot be reached.
+
+## Projects
+
+Each project must have a MAINTAINERS.md file with at least one maintainer. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Releases should be announced on the prometheus-users mailing list. Any new repositories should be first proposed on prometheus-team@ and following the voting procedures listed below. When a repository is no longer relevant, it should be moved to the prometheus-junkyard Github organization.
+
+## Decisionmaking
+
+### Team members 
+
+Team member status is given to those who have made ongoing contributions for at least 3 months to the Prometheus Project. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account. 
+
+New members may be proposed by any existing member by email to [prometheus-team][team] and voted on by [consensus](#consensus). If no consensus is reached, the new member is not accepted.
+
+Team members are added to the [GitHub organization][gh] and have commit rights to all projects. They should however respect the maintainers of each project.
+
+Team members may retire at any time by emailing [the team][team].
+
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
```

Comment:
the wordings were not clear on whether they would be _allowed_ to vote, but it would have been extremely ambiguous if someone were to vote against their own removal and the vote counted but not towards the quorum … so I think this is the best way to keep things straight.